.github/workflows/ci.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on: [push]
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        ruby: ['2.7', '3.0', '3.1']
+        activerecord: [6.1, 7.0]
```

Comment:
We should make sure all of these match. I don't remember what we have for Rails version specificity on Travis, but as I mentioned on that PR, the proposed `travis.yml` has a fixed Ubuntu release as well as patch versions on ruby.